### DISCUSSION TITLE
So how do you "force Classic Theme in 10/11"?

### DISCUSSION THREAD

**Discussion by Ulti-P-Uszer** [Abdelrhman-AK/WinPaletter#29] [Q&A] [answered]
So how do you "force Classic Theme in 10/11"?. I am using the ported classic theme on LTSC. Is using the ported classic theme (from here on GitHub) how you force it?

**Comment by Abdelrhman-AK** [comment, marked as answer]
You can force it by using SimpleClassicTheme: Search it in Github

**Reply by OrthodoxWindows** [reply]
I add that it is also possible using this tutorial (https://github.com/valinet/ExplorerPatcher/discussions/167 https://winclassic.boards.net/thread/990/windows-classic-theme-on-1903), and my scripts (https://github.com/OrthodoxWindows/ThemeSwitcherScripts https://winclassic.boards.net/thread/1221/themeswitcherscripts), which can be installed with a fairly similar method.
